aerospike-storage-backend/src/main/java/com/playtika/janusgraph/aerospike/AerospikeStoreManager.java

```diff
@@ -176,12 +200,21 @@ public void processAndDeleteTransaction(Value transactionId,
         } catch (AerospikeException e) {
             throw new PermanentBackendException(e);
         } finally {
-            releaseLocks(keysLocked);
-            deleteWalTransaction(transactionId);
+            releaseLocksAndDeleteWalTransaction(keysLocked, transactionId);
         }
     }
 
-    void releaseLocks(Set<Key> keysLocked) {
+    public void releaseLocksAndDeleteWalTransaction(Map<String, Map<Value, Map<Value, Value>>> locksByStore, Value transactionId) throws BackendException {
+        lockOperations.releaseLocks(locksByStore);
+        deleteWalTransaction(transactionId);
+    }
+
+    private void releaseLocksAndDeleteWalTransaction(Set<Key> keysLocked, Value transactionId) throws BackendException {
+        releaseLocks(keysLocked);
+        deleteWalTransaction(transactionId);
+    }
+
+    void releaseLocks(Set<Key> keysLocked) throws BackendException {
```

Comment:
you can inline the method

Reply:
it's overridden in test implementation: [FlakingAerospikeStoreManager](https://github.com/Playtika/aerospike-janusgraph-storage-backend/blob/fb0a86a83e6191d7f13b9bc779416edb70ed83ee/aerospike-storage-backend/src/test/java/com/playtika/janusgraph/aerospike/LocksGetReleasedAfterFailureTest.java#L185)